_maps/shuttles/emergency_humpback.dmm

```diff
@@ -0,0 +1,1314 @@
+/area/shuttle/escape)
+"bL" = (
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/turf/open/floor/mineral/plastitanium/red,
+/area/shuttle/escape/brig)
+"bS" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/obj/structure/chair/comfy/shuttle,
+/turf/open/floor/mineral/plastitanium/red,
+/area/shuttle/escape)
+"bT" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 8
+	},
+/turf/open/floor/mineral/plastitanium/red,
+/area/shuttle/escape)
+"cK" = (
+/turf/open/floor/iron/smooth_half/airless,
+/area/shuttle/escape)
+"df" = (
+/obj/machinery/door/airlock/external,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/obj/effect/turf_decal/siding/blue,
+/obj/effect/mapping_helpers/airlock/access/all/command/general,
+/obj/effect/mapping_helpers/airlock/cyclelink_helper_multi{
+	cycle_id = "evac3"
+	},
+/turf/open/floor/mineral/plastitanium/red,
+/area/shuttle/escape)
+"dl" = (
+/obj/structure/chair/sofa/right{
+	dir = 4
+	},
+/turf/open/floor/iron/smooth,
+/area/shuttle/escape)
+"dF" = (
+/obj/machinery/power/shuttle_engine/propulsion{
+	dir = 1
+	},
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/escape/brig)
+"dP" = (
+/obj/structure/table/reinforced/plastitaniumglass,
+/obj/effect/spawner/random/food_or_drink/booze,
+/turf/open/floor/mineral/plastitanium,
+/area/shuttle/escape)
+"ed" = (
+/obj/structure/table,
+/obj/item/storage/medkit/regular,
+/obj/item/storage/medkit/toxin{
+	pixel_x = 4;
+	pixel_y = 4
+	},
+/turf/open/floor/wood/large,
+/area/shuttle/escape)
+"ez" = (
+/obj/machinery/door/airlock/public/glass{
+	name = "Cyborg Bay"
+	},
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/turf/open/floor/mineral/plastitanium/red,
+/area/shuttle/escape)
+"eK" = (
+/obj/structure/window/reinforced/spawner/directional/south,
+/obj/machinery/power/shuttle_engine/heater{
+	dir = 1
+	},
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/escape)
+"eU" = (
+/obj/machinery/door/airlock/external/glass,
+/obj/effect/mapping_helpers/airlock/cyclelink_helper_multi{
+	cycle_id = "evac3"
+	},
+/turf/open/floor/mineral/plastitanium,
+/area/shuttle/escape)
+"eW" = (
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/obj/machinery/status_display/evac/directional/east,
+/turf/open/floor/mineral/plastitanium,
+/area/shuttle/escape)
+"fu" = (
+/obj/machinery/computer/emergency_shuttle{
+	dir = 1
+	},
+/turf/open/floor/mineral/plastitanium/red,
+/area/shuttle/escape)
+"fJ" = (
+/obj/effect/mapping_helpers/emergency_shuttle_event_override/random_pick{
+	events_override = list(/datum/shuttle_event/simple_spawner/carp/friendly = 6, /datum/shuttle_event/simple_spawner/carp/friendly_but_no_personal_space = 3, /datum/shuttle_event/simple_spawner/carp = 2, /datum/shuttle_event/simple_spawner/carp/magic = 1)
```

Comment:
Generally you'd make this a subtype with the set vars, but you should just put this on the shuttle template as I mentioned. 

I should also let you know anything but the friendly spawns (especially magic and friendly_but_no_personal_space) are incredibly disruptive, but I'm down if you are

Reply:
its funny though

Reply:
you make a good point